components/honor_x3

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "honor_x3",
+  "owner": ["@juacas"],
+  "manifest": "https://raw.githubusercontent.com/juacas/honor_x3/master/custom_components/honor_x3/manifest.json",
```

Comment:
`requests` is a core requirement of Home Assistant, you should not have this in your manifest file.

Reply:
Thanks. Now it is removed from manifest.